src/Api/Controllers/UsersController.cs

```diff
@@ -35,7 +36,7 @@ public UsersController(CmsDbContext dbContext, ILoggerFactory logger) : base(dbC
         /// <response code="409">Resource already exists</response>        
         /// <response code="503">Service unavailable</response>        
         /// <response code="401">User is denied</response>
-        [HttpPost("Invite")]
+        [HttpPost("/Api/Users/Invite")]
```

Comment:
This style of setting the URL is inconsistent to the other controllers... shouldn't we create a separate UsersController that has the Invite and GetUsers endpoints, as these are semantically different?